scripts/test-kic-2.x-upgrade.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+
+# ------------------------------------------------------------------------------
+# test-kick-2.x-upgrade.sh
+#
+# This script is temporary: in the timeframe between 1.x and 2.x KIC
+# releases this script was made to validate that upgrading to the
+# latest pre-release image was functional, once 2.0 releases fully
+# this script will be removed because all tests thereafter will be
+# running on that version and it's lineage.
+# ------------------------------------------------------------------------------
+
+# ------------------------------------------------------------------------------
+# Environment Variables
+# ------------------------------------------------------------------------------
+
+if [[ -z $TEST_ENV_NAME ]]
+then
+    TEST_ENV_NAME="kong-charts-tests"
+fi
+
+# ------------------------------------------------------------------------------
+# Shell Configuration
+# ------------------------------------------------------------------------------
+
+set -euo pipefail
+
+# ------------------------------------------------------------------------------
+# Kubernetes Cluster Setup
+# ------------------------------------------------------------------------------
+
+kubectl cluster-info --context kind-${TEST_ENV_NAME} 1>/dev/null
```

Comment:
Please run shellcheck on this file and fix all errors. Here it should complain about the variable expansion not being quoted.

Reply:
0caae597af65de6708c83a27f8b5a2c0a24687f0 resolves this, but I did you one better and make CI checks for this in 3409c9ddc068d7c3ccc956d15a00991287c4a862 :+1: